marketplace/pom.xml

```diff
@@ -1,41 +1,59 @@
-  <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.nuxeo.ecm.platform</groupId>
-    <artifactId>shibboleth-login-parent</artifactId>
-    <version>2.7.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
-  </parent>
+    <parent>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>shibboleth-login-parent</artifactId>
+        <version>2.7.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
 
-  <artifactId>marketplace-shibboleth-authentication</artifactId>
-  <packaging>zip</packaging>
-  <name>Shibboleth Authentication Marketplace Package</name>
+    <artifactId>marketplace-shibboleth-authentication</artifactId>
+    <packaging>zip</packaging>
+    <name>Shibboleth Authentication Marketplace Package</name>
 
-  <dependencies>
-    <!-- Additional bundles to be deployed by the MP -->
-    <dependency>
-      <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-platform-login-shibboleth</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.nuxeo.ecm.platform</groupId>
-      <artifactId>nuxeo-usermapper</artifactId>
-    </dependency>
-  </dependencies>
+    <dependencies>
+        <dependency>
+            <groupId>org.nuxeo.ecm.distribution</groupId>
+            <artifactId>nuxeo-nxr-server</artifactId>
+            <type>zip</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.nuxeo.ecm.distribution</groupId>
+            <artifactId>nuxeo-nxr-jsf-ui</artifactId>
+            <type>zip</type>
+            <scope>provided</scope>
+        </dependency>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.nuxeo.build</groupId>
-        <artifactId>ant-assembly-maven-plugin</artifactId>
-        <configuration>
-          <buildFiles>
-            <buildFile>${basedir}/src/main/assemble/assembly.xml</buildFile>
-          </buildFiles>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+        <!-- Additional bundles to be deployed by the MP -->
+        <dependency>
+            <groupId>org.nuxeo.ecm.platform</groupId>
+            <artifactId>nuxeo-platform-login-shibboleth</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.nuxeo.ecm.platform</groupId>
+            <artifactId>nuxeo-platform-login-shibboleth-jsf</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.nuxeo.ecm.platform</groupId>
+            <artifactId>nuxeo-usermapper</artifactId>
```

Comment:
You can remove that dependency, it is already a dependency of `org.nuxeo.ecm.platform:nuxeo-platform-login-shibboleth` so it will end up in the MP.

Reply:
I agree and now the use-mapper dependency is end up in the MP